### ISSUE TITLE
OutgoingRinger NPE on network failure

### ISSUE THREAD

**Issue by haffenloher** [WhisperSystems/Signal-Android#5353]
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

### Bug description
Got this crash today during the Redphone relay outage:
```
D/RedPhoneService(18728): java.lang.NullPointerException
D/RedPhoneService(18728): 	at org.thoughtcrime.redphone.audio.OutgoingRinger.onPrepared(OutgoingRinger.java:147)
D/RedPhoneService(18728): 	at android.media.MediaPlayer$EventHandler.handleMessage(MediaPlayer.java:1992)
D/RedPhoneService(18728): 	at android.os.Handler.dispatchMessage(Handler.java:99)
D/RedPhoneService(18728): 	at android.os.Looper.loop(Looper.java:153)
D/RedPhoneService(18728): 	at android.app.ActivityThread.main(ActivityThread.java:5299)
```
Happens [here](https://github.com/WhisperSystems/Signal-Android/blob/fa22fb755084b3c2d8064811d6be79ee0b617eac/src/org/thoughtcrime/redphone/audio/OutgoingRinger.java#L147), not sure how that's possible? Might be something for redphone wizard @cascheberg

### Steps to reproduce
- try to place a Signal call during the relay outage (setting phone to airplane mode somehow isn't enough, so I can't reproduce :/)
- get a "network failed" message
- crash

### Device info
 **Android version:** 4.2.2

 **Signal version:** 3.14.2

edit: link to the full log: https://gist.github.com/anonymous/db8a786f76f5400c58aa

**Comment by cascheberg** [comment]
This would also be fixed by #4886.

**Comment by agrajaghh** [comment]
fixed by #5357
